imports/ui/unit-wizard/unit-wizard.jsx

```diff
@@ -185,24 +185,24 @@ class UnitWizard extends Component {
                 disabled={inProgress}
                 onChange={this.createTextStateHandler('zipCode')}
               />
-              <RaisedButton
-                className='mt3'
-                fullWidth
-                primary
-                disabled={this.checkIsFormInvalid() || inProgress}
-                onClick={this.handleAddClicked}
-              >
-                {inProgress ? (
-                  <div className='absolute top-0 right-0 bottom-0 left-0'>
-                    <CircularProgress color='white' size={30} />
-                  </div>
-                ) : (
-                  <div className='f4 white'>Add Unit</div>
-                )}
-              </RaisedButton>
             </div>
           </div>
         </form>
+        <RaisedButton
+          className='mt3 bottom-0 right-0'
```

Comment:
Since you removed 'absolute', 'bottom-0' and 'right-0' are not needed either (they don't have any effect without 'absolute' anyway) .